Fixes #72 - added new template varaible

TakeitEasyAlan opened the pull request:
### Requirements

it now prefixes the variable in the serialization file correctly

### Description of the Change

added a templatedata variable with the layer name that was lower-cased to match sitecore's way f declaring varaibels

### Benefits

serialization file has the correct values.

TakeitEasyAlan committed:
Fixes #72 - added new template varaible

Saturate reviewed (approved):
Tested and works great. But maybe we should write a test for it before merge?

Saturate committed:
Add assert test for serialization.config

Makes sure that the content of the serialization.config fits whats
expected

#72

TakeitEasyAlan commented:
Thanks for adding the test @Saturate
